tests/filecheck/mlir-conversion/builtin_attrs.mlir

```diff
@@ -31,19 +44,19 @@
 
   "func.func"() ({}) {function_type = () -> (), value = true, sym_name = "true_attr"} : () -> ()
 
-  // CHECK: value = true
+  // CHECK: "value" = true
```

Comment:
I am confused about this change. Did this work before? Why should this be changed in this PR?

It feels like printing as xDSL should have this result, whereas printing with MLIR should be the old one :sweat_smile:

Reply:
I tried to described it in the PR message, but essentially this file was never running on the CI, and was actuallly not working.
So I fixed it to be able to test this PR. I'll fix the non-running issue on another PR though.
I put this change on another commit, so we won't have a single commit containing both changes (I'll not squash this time).